web/src/main/java/com/epam/rft/atsy/web/controllers/FileDownloadController.java

```diff
@@ -37,14 +37,13 @@
   @Autowired
   private CandidateService candidateService;
 
-
   @ResponseBody
   @RequestMapping(value = "/secure/candidate/fileDownload/{candidateId}", method = RequestMethod.GET)
   public Object downloadFile(@PathVariable("candidateId") Long candidateId, RedirectAttributes redirectAttrs)
       throws Exception {
 
     String cvFilename = candidateService.getCvFilenameById(candidateId);
-    if (cvFilename != null) {
+    if (cvFilename != null && !cvFilename.isEmpty()) {
```

Comment:
I would like to suggest this method: org.apache.commons.lang3.StringUtils.isNotEmpty